README.md

```diff
@@ -19,6 +19,171 @@ Things you may want to cover:
 
 * Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+* Deployment instructions </br>
 
-* ...
+## Endpoints
+URL: https://head-to-toe-be.herokuapp.com/ </br></br>
+The medical professionals endpoint current has two required parameters: type, and state.  
+The type parameter accepts two values: doctors and mhp ( mental health professionals ).  
+Currently the state parameter only accepts capitalized spelled out states.  
+Example requests:  
+
+GET `/medical_professionals?type=doctor, state=South Dakota`   
```

Comment:
Can you change the comma to '&' and the param 'state' to 'location'? I'm guessing at some point state was changed to location and the readme just wasn't caught up yet